doc/classes/RayCast3D.xml

```diff
@@ -37,7 +37,7 @@
 		<method name="force_raycast_update">
 			<return type="void" />
 			<description>
-				Updates the collision information for the ray. Use this method to update the collision information immediately instead of waiting for the next [code]_physics_process[/code] call, for example if the ray or its parent has changed state.
+				Updates the collision information for the ray. Use this method to update the collision information immediately instead of waiting for the next [code]_physics_process[/code] call, for example, if the ray or its parent has changed state.
```

Comment:
Here it's not clear to me if it's an improvement, as the added pause adds ambiguity about whether it's like "waiting for the next process call, for example" (it could be waiting for several things, and the next _physics_process call is one of them). I'd suggest rephrasing to avoid this.

Reply:
I fixed this, and the same thing in RayCast2D.